lambda-functions/jira-webhook-receiver/config.js.txt

```diff
@@ -8,13 +9,33 @@ module.exports = {
   },
   slack: {
     api_token: process.env.slack_api_token,
-    bot_name: 'jira-filter-slack-bot',
-    bot_emoji: ':bear:',
   },
-  rules: {
-    resolutionExpression: /fixed|done/i,
-    resolutionChannel: '#jira-resolution'
-    emptyComponentProjectExpression: /project5/i,
-    emptyComponentName: 'no-component',
-  }
+  rules: [
+    // add as many rules as needed
+    {
+      description: 'a rule description for logging',
+      // rules match on conditions in if
+      // conditions can be a string, regex or a function with an issue passed in
```

Comment:
a function that returns a boolean i guess

Reply:
+1